### DISCUSSION TITLE
Milestones containing issues from multiple repositories

### DISCUSSION THREAD

**Discussion by zstix** [community/community#2875] [Discussions] [answered]
It would be great if there was a way to add issues from multiple repositories to a single milestone.

Our team treats milestones like "epics" and the work often spans multiple repositories (i.e. a Gatsby site repo and a Gatsby theme repo). It would be really helpful if we could make issues in the relevant repositories (for in-repo visibility and easy linking to PRs) but have a single milestone to show a list of the work left to do.

It looks like there are already [third-party tools](https://codetree.com/) out there to achieve this, but I'd love to see this added in GitHub.

Somewhat related to #41 (multi-repository discussions).

**Comment by becca** [comment, marked as answer]
As mentioned in https://github.com/github/feedback/discussions/41#discussioncomment-119465 this repository is currently scoped to only be for Discussions and GitHub for mobile feedback at this time. I'll do my best to make sure this information is shared with the relevant teams though!

**Reply by zstix** [reply]
Thanks @becca! If y'all need any additional information (or you have an issue we can watch), just let me know!
